main/bamboo/bamboo.go

```diff
@@ -105,7 +106,12 @@ func initServer(conf *configuration.Configuration, conn *zk.Conn, eventBus *even
 	// Static pages
 	router.Use(martini.Static(path.Join(executableFolder(), "webapp")))
 
-	registerMarathonEvent(conf)
+	if conf.Marathon.UseEventStream {
+		// Listen events stream from Marathon
+		listenToEventStream(conf, eventSubAPI)
```

Comment:
Ah, stop :) `listenTo<...>` was used to be consistent with `listenToZookeeper`. `registerMarathonEvent` will register Bamboo in Marathon, but `listenToEventStream` will listen stream and will not register anything. So I will go with `listenToEventStream` if you don't mind